inc/healthcheck/namespace.php

```diff
@@ -39,9 +40,14 @@ function output_page( array $checks ) {
 		$format = 'json';
 	}
 
+	$json_response = [
+		'status' => $passed ? 'OK' : 'Failure!',
```

Comment:
I believe the idea was for this to be machine-readable, so should be a status code rather than human text:
```suggestion
		'status' => $passed ? 'ok' : 'failed',
```

Reply:
I missed this - fixed.